### PR TITLE
MKL 1D ReLU fix

### PR DESCRIPTION
**Pull request by ykim362**
This is a fix for #6615 .

### CODE REVIEW

File: src/operator/mkl/mkl_relu-inl.h [hunk 1]
```diff
@@ -197,7 +203,15 @@ class MKLReluOp : public Operator {
...
-    if (out_grad[activation::kOut].ndim() == 2) {
+    if (out_grad[activation::kOut].ndim() == 1) {
+      Shape<4> dshape = Shape4(out_grad[activation::kOut].shape_[0], 1, 1, 1);
+      m_out_grad = mkl_experimental_direct_get_with_shape<xpu, 4, DType>(
+        out_grad[activation::kOut], dshape, s);
+      m_out_data = mkl_experimental_direct_get_with_shape<xpu, 4, DType>(
+        out_data[activation::kOut], dshape, s);
+      m_in_grad = mkl_experimental_direct_get_with_shape<xpu, 4, DType>(
+        in_grad[activation::kData], dshape, s);
+    } else if (out_grad[activation::kOut].ndim() == 1) {
```
Comment (piiswrong) [inline src/operator/mkl/mkl_relu-inl.h]: == 2?

Comment (ykim362) [inline src/operator/mkl/mkl_relu-inl.h]: Yes, that's correct. I will update the PR. Thanks!

**Comment by mli** [comment]
@ykim362 please consider update the mklml path in the docker file when changing the mklml version. 

see the comment: https://github.com/dmlc/mxnet/blob/master/prepare_mkl.sh#L58 and #6876

**Comment by ykim362** [comment]
@mli I see. I will do that from the next update. Thanks!

**Pull request merged**